graylog2-web-interface/src/views/components/aggregationwizard/elementConfiguration/VisualizationConfiguration.tsx

```diff
@@ -99,15 +99,15 @@ const VisualizationConfiguration = () => {
       </Field>
       {isTimelineChart && supportsEventAnnotations && (
         <Field name="visualization.eventAnnotation">
-          {({ field: { name, value, onChange }, meta: { error } }) => (
+          {({ field: { name, value = false }, meta: { error } }) => (
             <Input id={`${name}-input`}
                    label="Show Event annotations"
                    error={error}
                    labelClassName="col-sm-11"
                    wrapperClassName="col-sm-1">
               <EventAnnotationCheckbox id={`${name}-input`}
                                        name={name}
-                                       onChange={onChange}
+                                       onChange={() => setFieldValue(name, !value)}
```

Comment:
Was there a specific reason thy you preferred `() => setFieldValue(name, !value)` over `() => onChange({ target: { value: !value, name } })`?

Reply:
No. It felt like a more explicit way, without having to dabble with creating a pseudo-event.